CrustTests/Company.swift

```diff
@@ -14,6 +14,35 @@ class Company {
 
 extension Company: AnyMappable { }
 
+enum CompanyCodingKey: Keypath {
```

Comment:
It's weird that this isn't named `KeyPath`
![image](https://cloud.githubusercontent.com/assets/2275582/26640716/1a569b18-45ee-11e7-917f-6a2338d0d40b.png)